edenia/eden-spend-explorer
# style(webapp): improve mobile view

### Improve mobile view

### What does this PR do?

- Resolve #261

### Steps to test

1. run the project and open it in your browser: localhost:3000
2. use mobile mode in your browser to see the responsive mode

#### CheckList
